src/lifecycle.js

```diff
@@ -42,7 +42,8 @@ export default {
       oldState.data !== newState.data ||
       oldState.columns !== newState.columns ||
       oldState.pivotBy !== newState.pivotBy ||
-      oldState.sorting !== newState.sorting
+      oldState.sorting !== newState.sorting ||
+      oldState.page !== newState.page
```

Comment:
I think this might be bad. Changing pages would trigger a dataModel update (which is expensive) when it doesn't really need to.